tensorflow_graphics/__init__.py

```diff
@@ -49,4 +49,4 @@
   __all__.remove("version")
 # pylint: enable=g-statement-before-imports,g-import-not-at-top
 
-__version__ = "HEAD"
```

Comment:
The version is auto-generated when setup.py is run. The reason for which it is "HEAD" is because in the source repo it's just the last commit that determines the version.

Reply:
I was wondering where I accidentally managed to change that...